jsx/Form.js

```diff
@@ -134,6 +134,159 @@ var FormElement = React.createClass({
   }
 });
 
+/**
+ * Search Component
+ * React wrapper for a searchable dropdown
+ */
+var SearchElement = React.createClass({
+
+  propTypes: {
+    name: React.PropTypes.string.isRequired,
+    options: React.PropTypes.object.isRequired,
+    id: React.PropTypes.string,
+    strictSearch: React.PropTypes.bool,
```

Comment:
Can you add a comment explaining what this is for? "strictSearch"  isn't obvious and I had to read all the way to the render function before I understood what it was.

(It wouldn't hurt to add comments explaining each of the other props either, but most of them are more obvious than strictSearch.